cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
>In the mean time, it doesn't make sense to me to display a name that can't be typed in to look up the node.

@aaronlehmann  What do you mean by saying "a name that can't be typed"? Currently it will always display the Hostname as the node's name because the Annotations.Name is an empty string so it uses the Hostname after doing the check.